src/components/About/About.js

```diff
@@ -17,16 +17,14 @@ export default class About extends React.Component {
 
 const styles = StyleSheet.create({
   background: {
-    color: 'white',
-    fontSize: 25,
-    textAlign: 'center',
-  }
+    backgroundColor: 'white',
+  },
 
   header: {
     color: '#9739E8',
     fontSize: 35,
     fontStyle: 'italic',
-    fontWeight: 'lighter',
+    fontWeight: '300',
```

Comment:
About/About.js: While the code works, there are a few aspects to address. Why is ImageBackground imported here? Why is there a white background here? The App.css already uses a specific background color. The `fontWeight: 'lighter'` actually compiles without error in React Native and describes the style in a clear manner, instead of `fontWeight: '300'`.